cerebrium/data-sharing-storage/persistent-storage.mdx

```diff
@@ -2,52 +2,96 @@
 title: "Persistent Storage"
 ---
 
-Cerebrium gives to access to persistent storage to store model weights, files and much more. This storage volume persists across your project, meaning that if
-you refer to model weights or a file created in a different deployment, you will be able to access it!
+Cerebrium gives to access to persistent storage to store model weights and files.
+This storage volume persists across your project, meaning that if
+you refer to model weights or files created in a different app but in the same project, you're able to access them.
 
-This allows you to load in model weights more efficiently as well as reduce the size of your deployment container images. Currently,
+This allows model weights to be loaded in more efficiently, as well as reduce the size of your app container images. Currently,
```

Comment:
Maybe explain this a bit better of when a user would use this. Store generations from application, model weights etc. Its kind of clear here